syft/workers/websocket_server.py

```diff
@@ -66,7 +71,7 @@ def __init__(
             msg = await websocket.recv()
             await self.broadcast_queue.put(msg)
 
-    async def _producer_handler(self, websocket):
+    async def _producer_handler(self, websocket: websockets.WebSocketCommonProtocol):
```

Comment:
I hope this is the right type to specify here, although looks a bit too specific. Please suggest if there is a better way to handle that.

Reply:
I think this is correct